setup.py

```diff
@@ -13,6 +13,7 @@
 longDescription = '\n' + longDescription
 
 setup(name='pynrrd',
+      python_requires='>=3.7',
```

Comment:
Added this https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires

Reply:
That’s fine with me. Seems like major Python packages long ago dropped support for <3.7